src/components/buttons/button.module.scss

```diff
@@ -0,0 +1,100 @@
+$default-bg-green: #b3e87d;
+$active-bg-green: #87cc42;
+$active-outlined-bg-green: #e2fbcc;
+$disabled-bg-green: #e2fbcc;
+
+$default-border-green: #87cc42;
+$disabled-border-green: #b3e87d;
+
+$default-label-color: #303240;
+$disabled-label-color: #c4c6cf;
+
+$default-text-color: #468c00;
+$active-text-color: #87cc42;
+$disabled-text-color: #b3e87d;
+
+.base {
+    min-height: 40px;
+    min-width: 40px;
+    line-height: 20px;
+    border-radius: 8px;
+    font-family: Open Sans;
```

Comment:
Same for Font-Family, disregard this for now, but when we will deliver 1st feature, we will create default theme for this